### ISSUE TITLE
[5-02] As the system, I want to automatically generate and send invoices once a membership is approved

### ISSUE THREAD

**Issue by westerandr** [sita-samoa/sita-membership#27]
# As the system, I want to automatically generate and send invoices once a membership is approved

Estimation: 6 hours

**Comment by ainsofs** [comment]
- [ ]     The document that is generated is public. Meaning anyone with the link can access the invoice . Better to make it private and only accessible to user, exec and coordinator roles.
- [ ]     If you lose the email there is no way easy way to access the invoice. Be good to have a link to it somewhere. Maybe an invoice section for the signup application.
- [ ]     The details on the invoice are from the home tab. Probably best to use the work details tab. We could have another ticket for the applicant to choose which details to be used for invoice (home or work)
- [ ]     The dollar sign after the number looks a bit weird. Maybe have the $ then the number in the invoice
- [ ] Update invoice to look similar to current Xero invoice
